src/main/java/com/fakng/fakngagrgtr/parser/apple/VacancyApple.java

```diff
@@ -0,0 +1,42 @@
+package com.fakng.fakngagrgtr.parser.apple;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Data
+@JsonIgnoreProperties({"localeInfo"})
+public class VacancyApple {
+
+    private String id;
+
+    private String jobSummary;
+
+    private List<LocationApple> locations;
+
+    private String positionId;
+
+    private String postingDate;
+
+    private String postingTitle;
+
+    private String postDateInGMT;
+
+    private String transformedPostingTitle;
+
+    private String reqId;
+
+    private Boolean managedPipelineRole;
+
+    private Integer standardWeeklyHours;
+
+    private TeamApple team;
+
+    private Boolean homeOffice;
+
+    public LocalDateTime getPostDateTime(){
```

Comment:
I don't think this is a good idea
If you have a DTO, then it's just for moving data here and there. It shouldn't be responsible for formatting it or anything. Please, move this method back to the class, where it's actually used